src/controllers/tasks/active.officers.details.controller.ts

```diff
@@ -0,0 +1,44 @@
+import { ActiveOfficerDetails } from "@companieshouse/api-sdk-node/dist/services/confirmation-statement";
+import { Session } from "@companieshouse/node-session-handler";
+import { NextFunction, Request, Response } from "express";
+import { getActiveOfficersDetailsData } from "../../services/active.officers.details.service";
+import { CORPORATE_DIRECTORS_PATH, CORPORATE_SECRETARIES_PATH, NATURAL_PERSON_DIRECTORS_PATH, NATURAL_PERSON_SECRETARIES_PATH } from "../../types/page.urls";
+import { urlUtils } from "../../utils/url";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
+    const submissionId = urlUtils.getSubmissionIdFromRequestParams(req);
+    const session: Session = req.session as Session;
+    const officers: ActiveOfficerDetails[] = await getActiveOfficersDetailsData(session, transactionId, submissionId);
+    let secretary = false;
```

Comment:
i'd rename these vars to containsSecretary, containsDirector etc...
I found it a bit confusing when i first read it as i thought we were checking what type the each officer was, but realised we are saying 'if the array contains a secretary then go to the secretary screen first'